releasenotes/notes/add-native-span-events-b04983442822d1b0.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Enables encoding span events as a top-level field in v0.4 payloads and introduces environment variable ``DD_TRACE_NATIVE_SPAN_EVENTS``.
```

Comment:
Might want to reword, since this doesn't "enable", it just adds the configuration for, which is disabled by default.

Might also want to say which version of the agent is required to us it, might be worth adding which version of the agent is required to the help text of the config option.